yaroslava_skomarovska/src/Calculator.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+
+public class Calculator {
```

Comment:
Seems this code shouldn't work. You should use scanner to read data inputed by user and then make calculations using this data
`int a = scanner.nextInt();`